app/assets/stylesheets/application.scss

```diff
@@ -64,15 +63,17 @@ img.img-responsive {
       margin: 0 auto $spacer-4;
     }
   }
-
 }
 
 .pquote-credit {
   margin-top: $spacer-3;
   font-size: 14px;
 }
 
-
+// User profile
+.bio-details span {
+  margin-right: $spacer-4;
```

Comment:
Instead of adding this as CSS, you can use a utility class: `mr-4` will give you the same thing.